module-2/backend/week_one.md

```diff
@@ -5,30 +5,80 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+ * GET views a page
+ * POST adds data
+ * PUT/PATCH updates data
+ * DELETE destroys data
+
 2. What is Sinatra?
+ * A server framework in which we can run Ruby code.
+
 4. What is MVC?
+ * Model(data)-View(output)-Controller(routes) set up for an application.  
+
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+ * This helps us share a common language/understanding with other developers, which allows us to collaborate and debug with more efficiency.
+ 
 6. What types of variables are accessible in our view templates without explicitly passing them?
+ * Instance variables
+ 
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
   
   ```ruby
   get '/horses' do
     erb :index
   end
   ```
-
+ * Just set `@count = 1` above the erb method, and `@count` is available in the `:index` file.
+ 
 8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed`?
+ * Append `, locals: name: "Mr. Ed"` to the erb line. 
+ 
 9. What's the purpose of ERB?
+ * It allows us to run Ruby code within HTML code. 
+ 
 10. Why do I need a development AND test database?
+ * We want to test that the database is CRUD-able, but we don't want to actually manipulate the 'production' database just for test data.
+ 
 11. What's responsive design?
+ * Building apps to account for variable view environments of a user's device.
+ 
 12. What is CRUD and why is it important?
+ * CRUD refers to apps/dbs that can Create, Read, Update, and Delete data. It is important because it allows the user to interact with a dynamic interface.
+ 
 13. What does HTTP stand for? 
+ * hypertext transfer protocol
+ 
 14. What are the two ways to interpolate Ruby in an ERB view template? What's the difference between these two ways?
+ * <% %> evaluates a ruby expression (e.g., an enumerable on a collection) with no visible output
+ * <%= %> displays the output of a ruby expression
+ 
 15. What's an ORM?
+ * object relational model. A database that creates normalized tables with relationships to each other.

```

Comment:
So, the ORM isn't the database itself, but a tool that we use to interact with databases in an 'objecty' way. For us right now, that's ActiveRecord (which is also the answer to 16). It makes it so that we don't have to worry too much about how the underlying database is working. We can just work with objects that inherit from the ORM and it takes care of the DB piece for us.